io.catenax.shared.business_partner_number/1.0.0/BusinessPartnerNumber.ttl

```diff
@@ -0,0 +1,89 @@
+#######################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#>.
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#>.
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#>.
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:samm:io.catenax.shared.business_partner_number:1.0.0#>.
+
+:BusinessPartnerNumber a samm:Aspect;
+    samm:preferredName "Shared Aspect for any business partner number"@en;
+    samm:description "This is a shared aspect for BPN with a regex."@en;
+    samm:properties (:bpnlProperty :bpnsProperty :bpnaProperty);
+    samm:operations ();
+    samm:events ().
+
+:bpnlProperty a samm:Property;
+    samm:preferredName "Business Partner Number Legal Entity Property"@en;
+    samm:description "Property based on a BPNA."@en;
+    samm:characteristic :BpnlTrait;
+    samm:exampleValue "BPNA0123456789ZZ".
```

Comment:
Your example value here is not matching with characteristic. Please have a look.